src/index.js

```diff
@@ -1 +1,41 @@
 /** @format */
+
+import './style.css';
+import store from './modules/storage.js';
+import {
+  addTask, displayTask, clearField, input,
+} from './modules/utility.js';
+
+const addBtn = document.querySelector('#addBtn');
+
+window.addEventListener('load', () => {
+  displayTask(store.tasks);
+});
+
+addBtn.addEventListener('click', (e) => {
+  e.preventDefault();
+
+  if (input.value === '') {
+    // console.log("All Fields are required");
+  } else {
+    addTask(store.tasks);
+    displayTask(store.tasks);
+    clearField();
+  }
+});
+
+document.addEventListener('click', (e) => {
+  if (e.target.classList.contains('trashBtn')) {
+    const removeTask = (index) => {
+      store.tasks = store.tasks.filter(
+        (task) => Number(task.index) !== Number(index),
+      );
+      localStorage.setItem('tasks', JSON.stringify(store.tasks));
+    };
+
+    removeTask(e.target.id);
+    displayTask(store.tasks);
+  }
```

Comment:
- [ ] Please remember, according to the project requirements, All changes to the To-Do List should be saved in local storage. After editing the task descriptions and refresh, the changes are not saved. please, kindly look into that.

![demo33](https://github.com/ghermaico135/webpack-todo/assets/65298048/a78714ea-67ed-4140-ad7f-885077f9cced)